functions-and-scope.js

```diff
@@ -72,6 +136,19 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 9
 
+// Laagste waarde is 0
+
+let currentMaxGrade = 0;
```

Comment:
Het is handiger om als default waarde de eerste waarde uit de array te pakken in plaats van `0`. Stel je voor dat je `const grades = [-2, -4, -1];` als waarde hebt, dan zou `currentMaxGrade` uiteindelijk de waarde `0` hebben terwijl die waarde helemaal niet eens in de array voorkomt 😉